packages/kit/src/runtime/client/client.js

```diff
@@ -91,18 +91,13 @@ export function create_client({ target, base, trailing_slash }) {
 	let current = {
 		branch: [],
 		error: null,
-		session_id: 0,
 		// @ts-ignore - we need the initial value to be null
 		url: null
 	};
 
 	let started = false;
 	let autoscroll = true;
 	let updating = false;
-	let session_id = 1;
```

Comment:
Not sure what this was for, seems unused so I removed it

Reply:
Back in the day, updating `$session` would cause an invalidation of all `load` functions that used it. This was related to that, and is no longer necessary